spring-cloud-previews/google-cloud-securitycenter-spring-starter/src/main/java/com/google/cloud/securitycenter/v1/spring/SecurityCenterSpringAutoConfiguration.java

```diff
@@ -157,6 +157,16 @@ public SecurityCenterSettings securityCenterSettings(
     }
     Retry serviceRetry = clientProperties.getRetry();
     if (serviceRetry != null) {
+      RetrySettings createSecurityHealthAnalyticsCustomModuleRetrySettings =
```

Comment:
Corresponds to client library changes from https://github.com/googleapis/google-cloud-java/pull/9331